force-app/main/default/classes/orderdetails.cls

```diff
@@ -0,0 +1,256 @@
+public  class orderdetails {
+    @AuraEnabled
+    public static List < OrderItem > fetchOrderProduct() {
+
+        return[Select id,Product2.Name,Product2Id,OrderId, Order.OrderNumber,Order.Under_QA__c,Order.On_Hand_Recieved__c, Order.Account.Name, Order.Quantity__c,Order.Received_Quantity__c,Order.Pending_Quantityy1__c, Order.Status,Product2.Rejected_Quantity__c from OrderItem WHERE Order.status!='Fully Received Under QA' order by Order.OrderNumber ASC];    
+    }
+    @AuraEnabled
+    public static List<Order> updateOrder(string ordervalue,integer valueRquantity) {
```

Comment:
Change variable name to strordervalue & intvalueRquantity. Change whereever required.